[DOCS] Adds what's new for 8.0 (#123884)
elastic/kibana: docs/user/whats-new.asciidoc

// tag::notable-highlights[]

[float]
=== Security features enabled by default

Authentication, authorization, and multi-tenancy

> with {kib} Spaces are on-by-default, with almost no configuration effort.

Reviewer: ```suggestion
with spaces are on-by-default, with almost no configuration effort.
```

> When you run {kib} for the first time, you'll be presented with a new interactive

Reviewer: ```suggestion
When you use {kib} for the first time, you're presented with a new interactive
```

> UI that will help you set up {kib} and connect it securely to {es}.

Reviewer: ```suggestion
UI that helps you securely set up and connect {kib} to {es}.
```

> Simply paste the enrollment token that you received when you ran

Reviewer: ```suggestion
Simply paste the enrollment token that you receive when you start
```

> {es}, and {kib} will set up all the previous security features

Reviewer: ```suggestion
{es}, and {kib} sets up all the security features
```

> and connect {es} over TLS.

Reviewer: ```suggestion
and connects to {es} over TLS.
```

[role="screenshot"]
image::images/highlights-security.png[Configure Elastic to get started]

> [float]

Reviewer 2: ```suggestion
[float]
[[index-pattern-rename]]
```
Perhaps we could add an anchor here? Then, when I link from the Cloud docs to explain the name change, the link will go right to this section.

=== Index patterns are now data views
Renaming index patterns to data views creates the foundation for building

> new and exciting features in {kib}. Your data views will work

Reviewer: ```suggestion
new and exciting features in {kib}. Your data views work
```

exactly as index patterns did in previous releases.

[float]
=== Maps updates vector tiles & basemaps

Elastic is offering full vector tile support, generally available, in {kib} *Maps*.
Enjoy faster performance and smooth navigation of your data at scale.

[role="screenshot"]
image::images/highlights-maps.png[Three vector tile examples in Maps]

The default basemaps in light and dark themes now use
the more understated desaturated basemap. This gives your data the most
visual attention.

[float]
=== Discover has field statistics (beta)

Enable the new {kibana-ref}/show-field-statistics.html[*Field statistics*] view
for a summary of all fields (or just selected fields) in your data.
You'll find interesting fields to visualize and learn more about the shape of your data.
The *Field statistics* view
is similar to the Index Data Visualizer in {ml-app}.

[role="screenshot"]
image::images/highlights-discover.gif[Fields statistics view in Discover]

[float]
=== Multi-layered time axis redesigned

> The time axis was upgraded across {kib} to solve key readability and

Reviewer: ```suggestion
To solve key readability and
```

> formatting issues. This redesign affects *Lens*, *TSVB*,

Reviewer: ```suggestion
formatting issues, the time axis was upgraded in *Lens*, *TSVB*,
```

aggregation-based visualizations, and *Discover*.

> This multi-layered format enables you to create easier to understand visualizations from your data

Reviewer: ```suggestion
The multi-layered format enables you to create easier to understand visualizations from your data.
```

[role="screenshot"]
image::images/highlights-time-axis.gif[Upgraded time axis]

[float]

> === Lens heatmaps support values mode

Reviewer: ```suggestion
=== Display data labels in heatmaps
```

> Enable *Show* on labels in the editor toolbar and,

Reviewer: ```suggestion
Enable *Show* on labels, and
```

> when there’s room, a responsive data label will appear in each heatmap cell.

Reviewer: ```suggestion
when there’s room, a data label appears in each heatmap cell.
```

[role="screenshot"]
image::images/highlights-lens.png[Show pop-up on heatmap cell]

[float]
=== Canvas editing toolbar renovated
This new design brings a set of editing controls in *Canvas*,
similar to what you see in *Dashboard*.

> Use this toolbar to quickly access the *Visualization Library* and

Reviewer: ```suggestion
Use the toolbar to quickly access the *Visualization Library* and
```

create elements and visualizations.

[role="screenshot"]
image::images/highlights-canvas.png[Editing toolbar in Canvas]

> Canvas also has these features similar to *Dashboard*.

Reviewer: ```suggestion
*Canvas* now also has the following new features:
```

> * You can create embedded visualizations in your *Canvas* workpads.

Reviewer: ```suggestion
* Create embedded visualizations in your *Canvas* workpads.
```

Promote favorites to the *Visualization Library* for re-use.

> * Editing a visualization from *Canvas* will return you to *Canvas* when you

Reviewer: ```suggestion
* For fewer clicks to author and edit existing workpads, editing visualizations in *Canvas* now returns you to *Canvas* when you
```

> click *Save and close*. Fewer clicks to author

Reviewer: ```suggestion
click *Save and close*. 
```

> and edit existing presentations.

Reviewer: ```suggestion
```

// tag::notable-highlights[]

[float]

> === Security on-by-default

Reviewer 3: ```suggestion
=== Security features are enabled and configured by default
```

Reviewer 3: We refer to this feature as "security on-by-default" internally, but I think that externally we want to indicate that security features are enabled and configured by default. That's the approach we're taking in the release highlight for the Elasticsearch release notes: https://github.com/elastic/elasticsearch/pull/83292/files#diff-6084df60d0e64f7e1c8b55063a278fb627eda6ace267952dba756f6bf4912bf7R19

Authentication, authorization, and multi-tenancy

> with {kib} Spaces are on-by-default, with almost no configuration effort.

Reviewer 3: ```suggestion
with {kib} Spaces are enabled by default, with almost no configuration effort.
```

When you run {kib} for the first time, you'll be presented with a new interactive

> UI that will help you set up {kib} and connect it securely to {es}.

Reviewer 3: ```suggestion
UI that will help you set up {kib} and connect it to a secured {es} cluster.
```

Simply paste the enrollment token that you received when you ran

> {es}, and {kib} will set up all the previous security features

Reviewer 3: ```suggestion
{es}, and {kib} will set up all {stack} security features
```

Reviewer 3: I think that referring to the Elastic Stack security features is meaningful here, which is how we discuss them in the Elasticsearch docs. Also, "previous security features" makes it sound like we're including something older, which might make users ask "What about current features?"
